cmd/helm/lint.go

```diff
@@ -97,10 +97,14 @@ func (l *lintCmd) run() error {
 
 	var total int
 	var failures int
+	rc := 0
 	for _, path := range l.paths {
 		if linter, err := lintChart(path, rvals, l.namespace, l.strict); err != nil {
 			fmt.Println("==> Skipping", path)
 			fmt.Println(err)
+			if err == errLintNoChart {
+				rc = 1
```

Comment:
It would probably be better to just return an error here so that cobra takes care of setting the non-zero return code. Setting the return code manually and calling `os.Exit` also has the side effect of causing any deferred functions not to run. I don't think that would affect this right now, but later down the road it could. What do you think?

Reply:
Right on, I asked some folks I know if they used Cobra to handle exit codes, but consensus was that wasn't the right tool to do so. I feel like it is though, I just don't know how to do that yet. 

I'll figure it out and push the solution.